src/qt/pivx/send.cpp

```diff
@@ -667,6 +674,11 @@ void SendWidget::onCoinControlClicked()
 
 void SendWidget::onShieldCoinsClicked()
 {
+    if (!walletModel->isSaplingEnforced()) {
+        inform(tr("Cannot perform shielded operations, v5 upgrade hasn't being enforced yet!"));
```

Comment:
change to either `hasn't been enforced` or `isn't being enforced`

Reply:
typo fixed